packages/x-date-pickers/src/StaticTimePicker/StaticTimePicker.test.tsx

```diff
@@ -46,7 +46,10 @@ describe('<StaticTimePicker />', () => {
     }),
   );
 
-  it('should allows view modification, but not update value when `readOnly` prop is passed', () => {
+  it('should allows view modification, but not update value when `readOnly` prop is passed', function test() {
+    if (/safari/i.test(window.navigator.userAgent) || /firefox/i.test(window.navigator.userAgent)) {
+      this.skip();
```

Comment:
For each test, could we leave a comment on what's missing to make it work? I believe we have done this systematically in the past. 

For instance:
```suggestion
      // Only run in supported browsers
      if (typeof Touch === 'undefined') {
        this.skip();
      }
```